testdata/list-clouds.md

```diff
@@ -0,0 +1,23 @@
+# Command "juju clouds"
+
+> See also: [add-cloud](https://docs.com/add-cloud), [update-cloud](https://docs.com/update-cloud), [remove-cloud](https://docs.com/remove-cloud), [update-credential](https://docs.com/update-credential)
+
+**Aliases:** list-clouds
+
+## Summary
+List clouds.
+
+## Usage
+```juju clouds [options] ```
+
+## Examples
+examples for clouds...
+
+## Details
+details for clouds...
+
+## Subcommands
+- [bar](https://docs.com/clouds/bar)
+- [baz](https://docs.com/clouds/baz)
+- [foo](https://docs.com/clouds/foo)
+
```

Comment:
I'll have to see concrete results before I get a good sense of this. But, at first impression, it looks good.